handlers/changes.js

```diff
@@ -170,6 +170,11 @@ var getChanges = function(feed) {
     body: { doc_ids: _.union(feed.requestedIds, feed.validatedIds) },
     method: 'POST'
   }, function(err, changes) {
+    if (feed.res.finished) {
+      // don't write to the response if it has already ended
```

Comment:
I'd add something like "In this case the change will be ignored, but it's ok because it'll be fetched by the next changes request that comes in"